spec.html

```diff
@@ -34410,6 +34410,7 @@ <h2>Syntax</h2>
       </emu-grammar>
       <emu-note>
         <p>The first two lines here are equivalent to CharacterClass.</p>
+        <p style="background: yellow">[It is less confusing not having to back out of the new syntax, only to dive back in. And if we wanted to use CharacterClass here, we would need to check for differences in static &amp; runtime semantics.]</p>
```

Comment:
Are these meant to be editors notes?

Reply:
I just put in what Markus put in that doc. (see https://docs.google.com/document/d/1Tbv3hfX9CxQtzH9r-JdxJsQZhmmDsidRUKKxg345JV0/edit ) We can remove them later before merge.

Reply:
Some of the yellow text is like code comments: Explaining why some piece of grammar or rule is chosen to be this way, or why an assert will always pass, or contrasting with something from the non-v behavior. Some of these might want to become NOTEs. Are there other ways we can add such comments in the spec?

(And some yellow text is truly temporary, with some leftover TODOs and such.)

Reply:
NOTEs are the appropriate way of adding non-normative explanation that you want to be visible to readers of the specification.

If you just want it to be visible to readers of the raw spec text, you can use an HTML comment.